ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/processor/TransactionTracer.java

```diff
@@ -136,7 +136,9 @@ public List<String> traceTransactionToFile(
                             stackedUpdater,
                             transaction,
                             transactionProcessor,
-                            new StandardJsonTracer(out, showMemory, true, true, false),
+                            // FIXME // REVIEWME do we want to change the JSON
+                            // debug calls to use EIP-7756 values for gas and opcode?
+                            new StandardJsonTracer(out, showMemory, true, true, false, true),
```

Comment:
Do we want to extend this to JSON-RPC calls?  Currently this only impacts CLI tools (and those have a CLI override flag).